examples/wide_angle_camera/GlutWindow.cc

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#if __APPLE__
+  #include <OpenGL/gl.h>
+  #include <OpenGL/OpenGL.h>
+  #include <GLUT/glut.h>
+#else
+  #include <GL/glew.h>
+  #include <GL/gl.h>
+  #include <GL/glut.h>
+#endif
+
+#if !defined(__APPLE__) && !defined(_WIN32)
+  #include <GL/glx.h>
+#endif
+
+#include <mutex>
+
+#include <ignition/common/Console.hh>
+#include <ignition/rendering/Image.hh>
+#include <ignition/rendering/Scene.hh>
+#include <ignition/rendering/WideAngleCamera.hh>
+
+#include "GlutWindow.hh"
+
+#define KEY_ESC 27
+#define KEY_TAB  9
+
+//////////////////////////////////////////////////
+unsigned int imgw = 0;
+unsigned int imgh = 0;
+
+std::vector<ir::CameraPtr> g_cameras;
+ir::CameraPtr g_camera;
+ir::CameraPtr g_currCamera;
+unsigned int g_cameraIndex = 0;
+ir::ImagePtr g_image;
+ignition::common::ConnectionPtr g_connection;
+
+bool g_initContext = false;
+
+#if __APPLE__
+  CGLContextObj g_context;
+  CGLContextObj g_glutContext;
+#elif _WIN32
```

Comment:
Add windows support ? here a reference https://github.com/ignitionrobotics/ign-rendering/pull/291

Reply:
adapted windows changes to the CMakeList and cpp files in this example. a826e9e